server/src/http/feature_refresher.rs

```diff
@@ -111,6 +112,7 @@ impl Default for FeatureRefresher {
             features_cache: Default::default(),
             engine_cache: Default::default(),
             persistence: None,
+            open: Default::default(),
```

Comment:
Personally a huge fan of explicit stuff, especially for primitives

```suggestion
            open: false,
```